examples/hotelling_law/requirements.txt

```diff
@@ -1,7 +1,3 @@
 mesa
-solara
-pandas
-matplotlib
-numpy
 scipy
-jupyter
\ No newline at end of file
+git+https://github.com/projectmesa/mesa-examples
```

Comment:
This line is not needed, and is confusing to be here.

Reply:
ah.. I was just going through other samples... I'm removing it... can you explain the context how other examples using it

Reply:
updated

Reply:
They should be removed. The experimental visualization used to live in the mesa-examples repo.